File: Golang/122. Best Time to Buy and Sell Stock II.go
func maxProfit(prices []int) int {
    ans := 0
    length := len(prices)
    if length < 2 {
        return 0
    } else {
        for i := 0; i < length-1; i++ {
            if prices[i] < prices[i+1] {
                ans += prices[i+1] - prices[i]
            }
        }
        return ans
    }
}

File: Python3/122. Best Time to Buy and Sell Stock II.py
class Solution:
    def maxProfit(self, prices):
        """
        :type prices: List[int]
        :rtype: int
        """
        ans = 0 # 48ms
        if len(prices) < 2: return ans
        for i in range(len(prices)-1):
            if prices[i] < prices[i+1]:
                ans += prices[i+1] - prices[i]
        return ans
